tests/acceptance/guid-file/registration-file-detail-test.ts

```diff
@@ -294,4 +296,109 @@ module('Acceptance | guid file | registration files', hooks => {
+        assert.dom('[data-test-file-language]').hasText('English',
+            'File metadata resource language is properly updated.');
+    });
+
+    test('View-only section does not update', async function(this: ThisTestContext, assert) {
+        await visit(`/--file/${this.file.id}`);
+
+        await click('[data-test-edit-metadata-button]');
+        // Update title
+        await fillIn('[data-test-title-field] > div > textarea', 'A New Title');
+        // Update description
+        await fillIn('[data-test-description-field] > div > textarea', 'A New Description');
+        // Update resource type
+        await selectChoose('[data-test-select-resource-type]', 'InteractiveResource');
+        // Update resource language
+        await selectChoose('[data-test-select-resource-language]', 'Latin');
+        // Cancel changes
+        await click('[data-test-cancel-metadata-button]');
+
+        assert.dom('[data-test-file-title]').doesNotHaveTextContaining('A New Title',
+            'Cancel metadata edit button properly working for title.');
+        assert.dom('[data-test-file-description]').doesNotHaveTextContaining('A New Description',
+            'Cancel metadata edit button properly working for description.');
+        assert.dom('[data-test-file-resource-type]').doesNotHaveTextContaining('InteractiveResource',
+            'Cancel metadata edit button properly working for resource type.');
+        assert.dom('[data-test-file-language]').doesNotHaveTextContaining('Latin',
+            'Cancel metadata edit button properly working for resource language.');
+    });
+
+    test('Error and canceling edit', async function(this: ThisTestContext, assert) {
+        setupOnerror((e: any) => assert.ok(e, 'Error is handled'));
+
+        const node = server.create('node', {
+            id: 'mtadt',
+            currentUserPermissions: [Permission.Read, Permission.Write],
+        });
+        const projectFile = server.create('file', {
+            target: node,
+            name: 'Test File',
+        });
+        const url = `/--file/${projectFile.id}`;
+
+        server.namespace = '/v2';
+        server.patch('/custom_file_metadata_records/:id', () => ({
+            errors: [{
+                detail: 'Could not patch metadata',
+                source: { pointer: 'points to nowhere' },
+            }],
+        }), 400);
+
+        const timer = sinon.useFakeTimers({ shouldAdvanceTime: true });
+
+        await visit(url);
+
+        await click('[data-test-edit-metadata-button]');
+        await fillIn('[data-test-title-field] > div > textarea', 'A New Title');
+        await click('[data-test-save-metadata-button]');
+        assert.dom('#toast-container', document as any)
+            .hasTextContaining(t('osf-components.file-metadata-manager.error-saving-metadata'));
+        await click('[data-test-cancel-metadata-button]');
+        await timer.tickAsync(5000); // skip until toast is gone
```

Comment:
Is it necessary for this test to skip ahead 5000 ms until the toast is gone? The assertion on line 384 should work regardless of whether the toast is there.

Reply:
I had the test fail previously without the delay. 5000ms had been the standard in the draft and revision test. I believe our toast is set for 3000ms and can check. If you think another value here would work better, let me know. However we had been guided towards consistency, and this was an opportunity to do so.

Reply:
This `tickAsync` was needed for those other tests cause there were multiple toast messages being tested, and we needed to wait for them to be gone in those tests. It shouldn't be needed here, as we are not testing multiple toast messages